packages/repository/README.md

```diff
@@ -168,7 +168,13 @@ coercion. The following types are supported out of box.
 - ArrayType
 - UnionType
 
-# Use Repository
+## Installation
+
+```
+$ npm install --save @loopback/repository
+```
+
+## Basic use
```

Comment:
@bajtos @superkhau @crandmck 
Ok, here is the part where I am confused ..
I thought the detailed explanation of `Repository` , `Model`, DataSource`, `Connector`, `Mixin` ... etc, would be in the OVERVIEW part .. Where do you want that? Clearly not in the Basic Use either !